StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/MaintainabilityRules/SA1413CodeFixProvider.cs

```diff
@@ -115,5 +121,11 @@ private static SyntaxNode RewriteAnonymousObjectInitializer(AnonymousObjectCreat
             var fixedInitializer = initializer.WithInitializers(newInitializerExpressions);
             return fixedInitializer;
         }
+
+        private static TextChange GetEnumMemberTextChange(Diagnostic diagnostic, SyntaxNode syntaxRoot)
+        {
+            var member = (EnumMemberDeclarationSyntax)syntaxRoot.FindNode(diagnostic.Location.SourceSpan);
```

Comment:
The `syntaxRoot.FindNode` call has already been done in the calling method, please reuse the result of that, as this is usually a quite expensive call.